airbyte-cdk/python/airbyte_cdk/models/__init__.py

```diff
@@ -5,5 +5,61 @@
 # are just wrappers on top of that stand-alone package which do some namespacing magic
 # to make the airbyte_protocol python classes available to the airbyte-cdk consumer as part
 # of airbyte-cdk rather than a standalone package.
-from .airbyte_protocol import *
-from .well_known_types import *
+from airbyte_protocol.models import (
```

Comment:
this is annoying, but mypy needs to know what types are available by inspecting the code.

I think there are few alternatives to this approach
1. Remove this namespacing magic and always import the models from airbyte-protocol. This would be a breaking change
2. Tell mypy to ignore attribute errors. I think this is a worse outcome because it leaves us open to a wide category of type errors

I'm comfortable with this approach because
1. The protocol doesn't change frequently
2. We can have a follow up item to automatically update this file when the airbyte-protocol is updated